README.md

```diff
@@ -1,310 +1,39 @@
-$ docker build -f ./chain/laika/Dockerfile -t MY_TAGNAME ./chain
-$ docker run ... MY_TAGNAME ...
-```
-
-### Running Tests on Contracts
-
-First, install the solidity compiler `solc` for compiling the contracts. You
-can follow the [official installation
-documentation](https://solidity.readthedocs.io/en/v0.5.8/installing-solidity.html). Make sure that you install the version `0.5.8`.
-From the root directory of the repository you can run the tests by calling `make test contracts`. This will create a virtual Python environment, install the
-dependencies, compile the contracts and run the tests.
-
-## access.laika.trustlines.foundation
-
-The Trustlines Foundation hosts a publically accessible node for the
-Laika Testnet.
-
-You can access it via the following URL:
-https://access.laika.trustlines.foundation
+The Trustlines Blockchain is a component of the [Trustlines Protocol](https://trustlines.foundation/protocol.html).
+The Trustlines Protocol is a set of rules to allow the transfer of value on top of existing trust
+relationships stored on a trustless infrastructure, here a blockchain.
+
+This repository is a mono-repository containing multiple different packages related to the Trustlines Blockchain (tlbc)
+considered as the mainnet for Trustlines, and to Laika, considered as the testnet for Trustlines.
+
+It contains:
+- blockchain client configurations and dockerfiles for tlbc and laika, found in `./chain/`
+- contracts implementation of the Trustlines auction, the bridge, the validator set, and the Trustlines Network Token
+found in `./contracts/`
+- python packages for deploying these contracts via cli found in the `./tools/` folder: 
+`auction-deploy`, `bridge-deploy`, and `validator-set-deploy`
+- the bridge client ran by bridge validators found in `./tools/bridge/`
+- the quickstart package used to easily set up a tlbc or Laika node found in `./tools/quickstart/`
+
+## Read more
+
+To learn more about the Trustlines Blockchain infrastructure or how to run a blockchain node, 
+refer to the [Trustlines Blockchain documentation](https://github.com/trustlines-protocol/blockchain/tree/master/docs/BlockchainInfrastructure.md)
+
+You can also learn more about 
+[the contracts](https://github.com/trustlines-protocol/blockchain/tree/master/contracts/README.md),
+[the auction-deploy tool](https://github.com/trustlines-protocol/blockchain/tree/master/tools/auction-deploy/README.md),
+[the bridge](https://github.com/trustlines-protocol/blockchain/tree/master/tools/bridge/README.md),
+[the bridge-deploy tool](https://github.com/trustlines-protocol/blockchain/tree/master/tools/bridge-deploy/README.md),
+[the quickstart](https://github.com/trustlines-protocol/blockchain/tree/master/tools/quickstart/README.md),
+[the validator-set-deploy tool](https://github.com/trustlines-protocol/blockchain/tree/master/tools/validator-set-deploy/README.md),
+by reading their corresponding readmes.
+
+## Installation
```

Comment:
I don't think our agreed on structure is relevant for this repository, since it is ambiguous who the user / developer of this repository is. Does he want to work on the bridge? deploy new contracts? run a blockchain node?

But I am fine writing it up as if the user would want to run the quickstart script.